### ISSUE TITLE
Throw if toString is not present

### ISSUE THREAD

**Issue by delvedor** [fastify/fast-json-stringify#40]
Example:
```js
const fastJson = require('fast-json-stringify')
const stringify = fastJson({
  type: 'object',
  properties: {
    field: { type: 'string' }
  }
})

console.log(stringify({ field: null }))
// => TypeError: Cannot read property 'toString' of null
```

What we want to do in the above case?
- `'null'`
- `''`

Other options?

**Comment by mcollina** [comment]
I think we should handle `null` and `undefined` better. It think they should both be `''`.

**Comment by delvedor** [comment]
Closed by #41
